cmd/root.go

```diff
@@ -35,6 +35,7 @@ func NewRootCMD() *cobra.Command {
 
 func setFlags(cmd *cobra.Command, v *viper.Viper) {
 	flags := cmd.Flags()
+	flags.Bool("allow-unsaved-entity", false, "Skip checking for file existing and any detection on the entity.")
```

Comment:
Let's use description:

`Normally files that don't exist on disk are skipped and not tracked. When this option is present, the file will be tracked even if it doesn't exist.`